package.json

```diff
@@ -35,8 +35,6 @@
 	"private": true,
 	"dependencies": {
 		"@babylonjs/viewer": "^8.2.0",
```

Comment:
I know it was here prior to this PR, but do we need this in the root @dawoodkhan82?

Reply:
Actually no, we should remove.

Reply:
done.